routers/admin-router.js

```diff
@@ -135,8 +135,11 @@ adminRouter.get('/api/forbidden', async (req, res) => {
 adminRouter.post('/api/forbidden', async (req, res) => {
   if (!req.user) return res.status(401).send({ error: 'User not logged in' });
   const result = await createForbiddenPair(req.session.activeGroup._id, req.body);
-  if (result.insertedId) return res.send({ success: 'Forbidden pair added', id: result.insertedId });
-  res.send({ error: 'Something went wrong' });
+  if (result.insertedId) {
+    return res.send({ success: 'Forbidden pair added', id: result.insertedId });
+  } else {
+    return res.send({ error: 'Forbidden pair already exists in the group' });
```

Comment:
This really means that ANY error will be returned with the message "pair already exists. Please rework this.